.github/CODEOWNERS

```diff
@@ -95,7 +95,7 @@ packages/react-make-styles/ @layershifter
 ### Fabric
 # common/
 packages/common-styles/src/ @phkuo
-common/_semanticSlots.scss  @phkuo
+common/_semanticSlots.scss @phkuo
```

Comment:
It might be good to add another person here in addition to Philip. Same with list stuff Thomas owns.

Reply:
Added `@microsoft/cxe-red` here.